+ source ../common/common_lib.sh
++ source /root/mugen-riscv/libs/locallibs/common_lib.sh
+++ python3 --version
+++ '[' 0 -eq 0 ']'
+++ source /root/mugen-riscv/libs/locallibs/common_lib_python.sh
+ main
++ type -t post_test
+ '[' -n function ']'
+ trap post_test EXIT INT HUP TERM
+ rpm -qa
+ grep expect
++ type -t config_params
+ '[' -n '' ']'
++ type -t pre_test
+ '[' -n function ']'
+ pre_test
+ LOG_INFO 'Start environmental preparation.'
+ message='Start environmental preparation.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Start environmental preparation.'
Sun Dec  4 09:53:36 2022 - INFO  - Start environmental preparation.
+ DNF_INSTALL rsync
+ pkgs=rsync
+ node=1
+ '[' -z '' ']'
+ tmpfile=
++ python3 /root/mugen-riscv/libs/locallibs/rpm_manage.py install --pkgs rsync --node 1 --tempfile ''
+ tmpfile2=/tmp/tmp1va5ecvj
+ '[' -z '' ']'
+ tmpfile=/tmp/tmp1va5ecvj
+ LOG_INFO 'End of environmental preparation!'
+ message='End of environmental preparation!'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'End of environmental preparation!'
Sun Dec  4 09:54:15 2022 - INFO  - End of environmental preparation!
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ LOG_INFO 'Start to run test.'
+ message='Start to run test.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Start to run test.'
Sun Dec  4 09:54:16 2022 - INFO  - Start to run test.
+ test_execution rsyncd.socket
+ service=rsyncd.socket
++ date '+%Y-%m-%d %T'
+ log_time='2022-12-04 09:54:16'
+ test_restart rsyncd.socket
+ service=rsyncd.socket
+ systemctl restart rsyncd.socket
Failed to restart rsyncd.socket: Unit rsyncd.socket not found.
+ CHECK_RESULT 5 0 0 'rsyncd.socket restart failed'
+ actual_result=5
+ expect_result=0
+ mode=0
+ error_log='rsyncd.socket restart failed'
+ '[' -z 5 ']'
+ '[' 0 -eq 0 ']'
+ test 5x '!=' 0x
+ test -n 'rsyncd.socket restart failed'
+ LOG_ERROR 'rsyncd.socket restart failed'
+ message='rsyncd.socket restart failed'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'rsyncd.socket restart failed'
Sun Dec  4 09:54:17 2022 - ERROR - rsyncd.socket restart failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 34'
+ message='../common/common_lib.sh line 34'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 34'
Sun Dec  4 09:54:17 2022 - ERROR - ../common/common_lib.sh line 34
+ return 0
+ SLEEP_WAIT 5
+ wait_time=5
+ cmd=
+ mode=1
+ python3 /root/mugen-riscv/libs/locallibs/sleep_wait.py --time 5 --cmd '' --mode 1
+ systemctl status rsyncd.socket
+ grep 'Active: active'
Unit rsyncd.socket could not be found.
+ CHECK_RESULT 1 0 0 'rsyncd.socket restart failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='rsyncd.socket restart failed'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'rsyncd.socket restart failed'
+ LOG_ERROR 'rsyncd.socket restart failed'
+ message='rsyncd.socket restart failed'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'rsyncd.socket restart failed'
Sun Dec  4 09:54:24 2022 - ERROR - rsyncd.socket restart failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 37'
+ message='../common/common_lib.sh line 37'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 37'
Sun Dec  4 09:54:25 2022 - ERROR - ../common/common_lib.sh line 37
+ return 0
+ systemctl stop rsyncd.socket
Failed to stop rsyncd.socket: Unit rsyncd.socket not loaded.
+ CHECK_RESULT 5 0 0 'rsyncd.socket stop failed'
+ actual_result=5
+ expect_result=0
+ mode=0
+ error_log='rsyncd.socket stop failed'
+ '[' -z 5 ']'
+ '[' 0 -eq 0 ']'
+ test 5x '!=' 0x
+ test -n 'rsyncd.socket stop failed'
+ LOG_ERROR 'rsyncd.socket stop failed'
+ message='rsyncd.socket stop failed'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'rsyncd.socket stop failed'
Sun Dec  4 09:54:26 2022 - ERROR - rsyncd.socket stop failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 39'
+ message='../common/common_lib.sh line 39'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 39'
Sun Dec  4 09:54:27 2022 - ERROR - ../common/common_lib.sh line 39
+ return 0
+ SLEEP_WAIT 5
+ wait_time=5
+ cmd=
+ mode=1
+ python3 /root/mugen-riscv/libs/locallibs/sleep_wait.py --time 5 --cmd '' --mode 1
+ systemctl status rsyncd.socket
+ grep 'Active: inactive'
Unit rsyncd.socket could not be found.
+ CHECK_RESULT 1 0 0 'rsyncd.socket stop failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='rsyncd.socket stop failed'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'rsyncd.socket stop failed'
+ LOG_ERROR 'rsyncd.socket stop failed'
+ message='rsyncd.socket stop failed'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'rsyncd.socket stop failed'
Sun Dec  4 09:54:34 2022 - ERROR - rsyncd.socket stop failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 42'
+ message='../common/common_lib.sh line 42'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 42'
Sun Dec  4 09:54:35 2022 - ERROR - ../common/common_lib.sh line 42
+ return 0
+ systemctl start rsyncd.socket
Failed to start rsyncd.socket: Unit rsyncd.socket not found.
+ CHECK_RESULT 5 0 0 'rsyncd.socket start failed'
+ actual_result=5
+ expect_result=0
+ mode=0
+ error_log='rsyncd.socket start failed'
+ '[' -z 5 ']'
+ '[' 0 -eq 0 ']'
+ test 5x '!=' 0x
+ test -n 'rsyncd.socket start failed'
+ LOG_ERROR 'rsyncd.socket start failed'
+ message='rsyncd.socket start failed'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'rsyncd.socket start failed'
Sun Dec  4 09:54:36 2022 - ERROR - rsyncd.socket start failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 44'
+ message='../common/common_lib.sh line 44'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 44'
Sun Dec  4 09:54:36 2022 - ERROR - ../common/common_lib.sh line 44
+ return 0
+ SLEEP_WAIT 5
+ wait_time=5
+ cmd=
+ mode=1
+ python3 /root/mugen-riscv/libs/locallibs/sleep_wait.py --time 5 --cmd '' --mode 1
+ systemctl status rsyncd.socket
+ grep 'Active: active'
Unit rsyncd.socket could not be found.
+ CHECK_RESULT 1 0 0 'rsyncd.socket start failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='rsyncd.socket start failed'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'rsyncd.socket start failed'
+ LOG_ERROR 'rsyncd.socket start failed'
+ message='rsyncd.socket start failed'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'rsyncd.socket start failed'
Sun Dec  4 09:54:43 2022 - ERROR - rsyncd.socket start failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 47'
+ message='../common/common_lib.sh line 47'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 47'
Sun Dec  4 09:54:44 2022 - ERROR - ../common/common_lib.sh line 47
+ return 0
+ test_enabled rsyncd.socket
+ service=rsyncd.socket
++ systemctl is-enabled rsyncd.socket
Failed to get unit file state for rsyncd.socket: No such file or directory
+ state=
+ '[' '' == enabled ']'
+ '[' '' == disabled ']'
+ '[' '' == masked ']'
+ '[' '' == static ']'
+ LOG_INFO 'Unit is indirect, ignoring.'
+ message='Unit is indirect, ignoring.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Unit is indirect, ignoring.'
Sun Dec  4 09:54:45 2022 - INFO  - Unit is indirect, ignoring.
+ journalctl --since '2022-12-04 09:54:16' -u rsyncd.socket
+ grep -i 'fail\|error'
+ grep -v -i 'DEBUG\|INFO\|WARNING'
+ CHECK_RESULT 1 0 1 'There is an error message for the log of rsyncd.socket'
+ actual_result=1
+ expect_result=0
+ mode=1
+ error_log='There is an error message for the log of rsyncd.socket'
+ '[' -z 1 ']'
+ '[' 1 -eq 0 ']'
+ test 1x == 0x
+ return 0
+ test_reload rsyncd.socket
+ service=rsyncd.socket
+ systemctl start rsyncd.socket
Failed to start rsyncd.socket: Unit rsyncd.socket not found.
+ systemctl reload rsyncd.socket
+ grep 'Job type reload is not applicable'
+ CHECK_RESULT 1 0 0 'Job type reload is not applicable for unit rsyncd.socket'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='Job type reload is not applicable for unit rsyncd.socket'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'Job type reload is not applicable for unit rsyncd.socket'
+ LOG_ERROR 'Job type reload is not applicable for unit rsyncd.socket'
+ message='Job type reload is not applicable for unit rsyncd.socket'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Job type reload is not applicable for unit rsyncd.socket'
Sun Dec  4 09:54:46 2022 - ERROR - Job type reload is not applicable for unit rsyncd.socket
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 91'
+ message='../common/common_lib.sh line 91'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 91'
Sun Dec  4 09:54:47 2022 - ERROR - ../common/common_lib.sh line 91
+ return 0
+ systemctl status rsyncd.socket
+ grep 'Active: active'
Unit rsyncd.socket could not be found.
+ systemctl status rsyncd.socket
+ grep 'inactive (dead)'
Unit rsyncd.socket could not be found.
+ return 1
+ LOG_INFO 'End of the test.'
+ message='End of the test.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'End of the test.'
Sun Dec  4 09:54:48 2022 - INFO  - End of the test.
+ CASE_RESULT 0
+ case_re=0
+ test -z 7
+ test 7 -gt 0
+ LOG_ERROR 'failed to execute the case.'
+ message='failed to execute the case.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'failed to execute the case.'
Sun Dec  4 09:54:49 2022 - ERROR - failed to execute the case.
+ exit 7
+ post_test
+ LOG_INFO 'start environment cleanup.'
+ message='start environment cleanup.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'start environment cleanup.'
Sun Dec  4 09:54:50 2022 - INFO  - start environment cleanup.
+ systtemctl stop rsyncd.socket
oe_test_socket_rsyncd.sh: line 37: systtemctl: command not found
+ DNF_REMOVE
+ node=1
+ pkg_list=
+ mode=0
+ [[ -z /tmp/tmp1va5ecvj ]]
+ '[' 0 -ne 0 ']'
+ '[' 1 == 0 ']'
+ python3 /root/mugen-riscv/libs/locallibs/rpm_manage.py remove --node 1 --pkgs '' --tempfile /tmp/tmp1va5ecvj
+ '[' 0 -ne 0 ']'
+ LOG_INFO 'Finish environment cleanup!'
+ message='Finish environment cleanup!'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Finish environment cleanup!'
Sun Dec  4 09:55:03 2022 - INFO  - Finish environment cleanup!
